crowbar_framework/app/controllers/installers_controller.rb

```diff
@@ -42,27 +46,42 @@ def status
   # Provides the restful api call for
   # /installer/start 	POST 	triggers installation
   def start
-    if Crowbar::Installer.successful?
-      respond_to do |format|
-        format.json { render json: Crowbar::Installer.status }
-        format.html { redirect_to installer_url }
+    header = :ok
+    msg, msg_type = "", :alert
```

Comment:
Style/ParallelAssignment: Do not use parallel assignment. (https://github.com/bbatsov/ruby-style-guide#parallel-assignment)